### ISSUE TITLE
[SCREENREADER]: GIBCT® VETTEC - Learn more buttons would benefit from ARIA-LABELs to provide more context

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#1252]
## Description

The `Learn more` buttons rely on proximity to a label or heading to answer the question "Learn more about what?". This can be improved for assistive devices and screen readers by adding `aria-label` attributes to the buttons. Example screenshots attached.

/cc @peggygannon for Content team guidance as needed

## Point of Contact

**VFS Point of Contact:** _Trevor, Peggy_

## Acceptance Criteria

* As a screen reader user, I want to hear contextual aria-labels read aloud. This text could sound like "Learn more about GI Bill benefits" or "Learn more about cumulative Post-9/11 active duty service".
* The text "Learn more" should be included in each `aria-label`, ideally as the first text. See the second WCAG guidance link for more on this point.

## Environment

* Windows 10
* Chrome
* JAWS
* https://staging.va.gov/gi-bill-comparison-tool/
* https://staging.va.gov/gi-bill-comparison-tool/ VET TEC school profile

## WCAG or Vendor Guidance (optional)

* [Name, Role, Value: Understanding SC 4.1.2](https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-rsv.html)
* [Understanding Success Criterion 2.5.3: Label in Name](https://www.w3.org/WAI/WCAG21/Understanding/label-in-name.html)

## PRs
https://github.com/department-of-veterans-affairs/vets-website/pull/10878

## Screenshots or Trace Logs

![Screen Shot 2019-08-22 at 4.37.04 PM.png](https://images.zenhubusercontent.com/5ac217b74b5806bc2bcd3fc8/e508e6a0-62ab-4d3f-b5c0-b128027046ec)

---
![Screen Shot 2019-08-22 at 4.37.19 PM.png](https://images.zenhubusercontent.com/5ac217b74b5806bc2bcd3fc8/4a172926-3237-4c6d-afbc-07b5eb73564a)

**Comment by 1Copenut** [comment]
@desireeturner && @ksantiagomorgan6 Can we include any Colmery 107 additional `Learn more` links in this work? The VA 508 office came back with it as a recommended improvement, so I figure we should capture all of the links at one time if that fits into your sprint scope.

**Comment by desireeturner** [comment]
@1Copenut we have other stories to cover the other Learn More links. Please take a moment to review these and let me know if it doesn't cover the 508 office feedback: 
- CT: Update Aria Label Text for Landing Page Learn More Links #19778
- CT: Update Aria Label Text for Search Results Learn More Links #19780
- CT: Update Aria Label Text for Profile Page Learn More Links #19781
- CT: Update Aria Label Text for VETTEC Profile Page Learn More Links #19782

**Comment by 1Copenut** [comment]
@desireeturner I reviewed all of the tickets and left one comment on the PR. I'm good for your team to close this item when the last update is made.

**Comment by desireeturner** [comment]
@1Copenut am I correct to assume you're referring to this PR? https://github.com/department-of-veterans-affairs/vets-website/pull/10911

cc @sonntag-adam

**Comment by 1Copenut** [comment]
@desireeturner Yes, that PR is the one I reviewed for this ticket.
